navigation/navigation.css

```diff
@@ -0,0 +1,104 @@
+@import url(../utilities/reset.css);
+@import url(../text\ utilities/text.css);
+.navbar {
+  display: flex;
+  align-items: center;
+  position: fixed;
+  top: 0;
+  width: 100%;
+  padding: 1rem 2rem;
+  border-color: var(--pure-white);
+  border-bottom-style: outset;
+}
+
+.navbar__header {
+  margin-left: 5rem;
+}
+
+.navbar__list {
+  list-style: none;
+  max-width: 100%;
+  display: flex;
+  align-items: center;
+  padding: 1rem;
+  gap: 2rem;
+  padding: 1rem;
+  margin-left: auto;
+  margin-right: 5rem;
+}
+
+.navbar__list__items {
+  cursor: pointer;
+  font-size: 1.8rem;
+}
+
+.hamburger {
+  font-size: 2.5rem;
+  cursor: pointer;
+  margin-right: 1rem;
+}
+
+
+@media only screen and (max-width: 768px) {
+  .hamburger {
+    display: flex;
+  }
+  .hamburger__list {
+    width: 18rem;
+    height: auto;
+    border-right: 1px solid var(--dark);
+    border-bottom: 1px solid var(--dark);
```

Comment:
Please use shorthand property for border

Reply:
Could you please help me with this, I want to apply border only at right and bottom.

Reply:
```
border: solid green; 
border-width: 2px 0 2px 2px; /* top, right, bottom, left */
```

Reply:
Thanks, for helping out, I have changed the properties for border and margin, could you please review it again